services/index/src/api.js

```diff
@@ -120,6 +127,10 @@ builder.declare({
   ].join('\n'),
 }, async function(req, res) {
   let namespace = req.params.namespace || '';
+  await req.authorize({
+    namespace,
+    namespaceGiven: namespace !== '',
```

Comment:
it looks like `namespaceGiven` isn't actually used in the scope expression. Am I understanding this correctly?